solution_runner/commands/defaults_and_choices.py

```diff
@@ -0,0 +1,13 @@
+from datetime import datetime
+
+
+def get_default_year() -> int:
+    """
+    :return: default year which is the current year if it's December, last year otherwise
+    """
+    today = datetime.today()
+    current_year = today.year
+    if today.month == 12:
```

Comment:
12 is a magic number, store it in a constant

Reply:
Thanks, changed in b608be48.